frontend/server/src/Controllers/Course.php

```diff
@@ -4239,7 +4239,7 @@ public static function getCourseTabsForTypeScript(
             'templateProperties' => [
                 'payload' => [
                     'courses' => $courses,
-                ],
+Lis                ],
```

Comment:
Esto hay que dejarlo como estaba:
```suggestion
                ],
```